assembl/static2/css/_assembl_base_styles.scss

```diff
@@ -10,16 +10,18 @@
 @import 'components/doughnut';
 @import 'components/dropdown';
 @import 'components/footer';
-@import 'components/loader';
 @import 'components/goUp';
+@import 'components/loader';
 @import 'components/navbar';
+@import 'components/overflowMenu';
```

Comment:
The overflow menu, the ellipsis "..." on a post to show for now the edit and delete actions.